ci/taos/checker-issue-comment.sh

```diff
@@ -27,16 +27,16 @@
 source ./config/config-environment.sh
 source ./common/api_collection.sh
 
-# check if input argument is correct. 
+# Check if an input argument is correct.
 if [[ $1 == "" ]]; then
-    printf "[DEBUG] ERROR: Please, use a correct issue number.\n"
+    printf "[DEBUG] ERROR: Please use a correct issue number.\n"
     exit 1
 fi
 
 input_issue=$1
 
-# --------------------------- Report module: submit check result to github-website --------------
-# execute automatic comment on new issue.
+# --------------------------- Report module: submit a check result to a github website --------------
+# Comment an appropriate reply automatically whenever new issue is created.
```

Comment:
IMHO, 'comment' is an intransitive verb that does not have an object.

- Post an appropriate comment  automatically...
- Submit an appropriate comment  automatically...

Just suggestion:)

Reply:
Thanks. Applied.